### PR TITLE
librsvg 2.54.4: rebuild against libxml 2.13

### PR DESCRIPTION
**Pull request by skupr-anaconda**
**Destination channel:** main

### Links

- [PKG-5281](https://anaconda.atlassian.net/browse/PKG-5281) 

[PKG-5281]: https://anaconda.atlassian.net/browse/PKG-5281?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by skupr-anaconda** [comment]
There are a lot of errors on `win64`:
```
Executing transaction: ...working... g_module_open() failed for C:\b\abs_c9tge87qsw\croot\librsvg_1721992222047\_h_env\Library\lib\gdk-pixbuf-2.0\2.10.0\loaders\pixbufloader-ani.dll: 'C:\b\abs_c9tge87qsw\croot\librsvg_1721992222047\_h_env\Library\lib\gdk-pixbuf-2.0\2.10.0\loaders\pixbufloader-ani.dll': The specified module could not be found.

g_module_open() failed for C:\b\abs_c9tge87qsw\croot\librsvg_1721992222047\_h_env\Library\lib\gdk-pixbuf-2.0\2.10.0\loaders\pixbufloader-bmp.dll: 'C:\b\abs_c9tge87qsw\croot\librsvg_1721992222047\_h_env\Library\lib\gdk-pixbuf-2.0\2.10.0\loaders\pixbufloader-bmp.dll': The specified module could not be found.

g_module_open() failed for C:\b\abs_c9tge87qsw\croot\librsvg_1721992222047\_h_env\Library\lib\gdk-pixbuf-2.0\2.10.0\loaders\pixbufloader-gif.dll: 'C:\b\abs_c9tge87qsw\croot\librsvg_1721992222047\_h_env\Library\lib\gdk-pixbuf-2.0\2.10.0\loaders\pixbufloader-gif.dll': The specified module could not be found.
```

**Comment by skupr-anaconda** [comment]
And another error:
```
cl ..\gdk-pixbuf-loader\io-svg.c

Microsoft (R) C/C++ Optimizing Compiler Version 19.29.30154 for x64

Copyright (C) Microsoft Corporation.  All rights reserved.

g-ir-cpp-kdkublcs.c

Command '['C:\\b\\abs_c9tge87qsw\\croot\\librsvg_1721992222047\\work\\win32\\tmp-introspectb04vyvir\\Rsvg-2.0.exe', '--introspect-dump=C:\\b\\abs_c9tge87qsw\\croot\\librsvg_1721992222047\\work\\win32\\tmp-introspectb04vyvir\\functions.txt,C:\\b\\abs_c9tge87qsw\\croot\\librsvg_1721992222047\\work\\win32\\tmp-introspectb04vyvir\\dump.xml']' returned non-zero exit status 3221225781.

NMAKE : fatal error U1077: 'C:\b\abs_c9tge87qsw\croot\librsvg_1721992222047\_build_env\python.EXE' : return code '0x1'

Stop.
```

**Review by JamesRobertsonGames — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -64,13 +64,13 @@ requirements:
     - {{ cdt('libxrender-devel') }}      # [linux]
   host:
     - gobject-introspection 1.*
-    - glib 2.*
-    - gettext   # [osx]
+    - glib {{ glib }}
+    - gettext 0.21.0 # [osx]
     - harfbuzz 4.3.0
```
Comment (danpetry) [inline recipe/meta.yaml]: a lot of these deps are specified in the root cbc.yaml, please use `{{ harfbuzz }}` etc (or nothing, although I prefer to be explicit)

**Review by danpetry — APPROVED** [review]
minor nit

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: Fixed

**Pull request merged**